pytgcalls/pytgcalls/mtproto/telethon_bridge.py

```diff
@@ -169,7 +169,7 @@ def register_update_handlers(self):
 
     async def resolve_and_set_join_as(self, join_as):
         if join_as is None:
-            self.join_as = await self.get_and_set_self_peer()
+            self.join_as = self.full_chat.groupcall_default_join_as
```

Comment:
There is should be subcondition… checking default join as on None value. And if it None use get_and_set_self_peer. Otherwise just use default join as

Reply:
I don't think that will be none but oki :)

Reply:
> I don't think that will be none

It can be None for voice chat that you don’t visit. And don’t select default join as by sending SaveDefaultGroupCallJoinAsRequest  request